.circleci/config.yml

```diff
@@ -36,7 +44,7 @@ jobs:
           paths:
             - ./emsdk/emsdk
             - ~/.ccache
-          key: v1-emsdk-{{ checksum "emsdk/Makefile" }}-v8-{{ .BuildNum }}
+          key: v1-emsdk-{{ checksum "emsdk/Makefile" }}-v11-{{ .BuildNum }}
```

Comment:
This shift is because it took a few iterations of rebuilding emsdk to find a sufficiently large `NUM_PARAMS`: now it's 61 (as opposed to 32 before) as it's required by scipy's fitpack (~47) and odrpack (61). I'm not sure if it can have adverse effect (e.g. on performance).